backend/Controllers/auth/signup.js

```diff
@@ -0,0 +1,122 @@
+const bcrypt = require('bcryptjs');
+const {pool} = require('../db');
+const email = require('../../utils/email');
+const ajv = require('../../Schema');
+const {signupSchema} = require('../../Schema/auth');
+
+const isUsed = (username) => {
```

Comment:
Don't use arrow functions, use normal functions with proper JSDoc comments.

Reply:
Address this comment please

Reply:
> Don't use arrow functions, use normal functions with proper JSDoc comments.

Sir there is nothing wrong with using arrow functions right

Reply:
Normal functions are easier to debug.
https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/
Avoid using arrow functions especially when you are declaring functions only.

Reply:
> Normal functions are easier to debug.
> https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/
> Avoid using arrow functions especially when you are declaring functions only.

ok